.github/workflows/release-dev.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Development App
+
+on:
+  release:
+    types: [created]
```

Comment:
The trigger here should be a commit on the main branch for dev releases. Only GitHub releases should trigger a prod deployment

Reply:
Thanks for catching this